holoviews/element/stats.py

```diff
@@ -0,0 +1,125 @@
+import param
+import numpy as np
+
+from ..core.dimension import Dimension, Dimensioned
+from .chart import Chart, Scatter
+
+
+class _StatisticsElement(Chart):
```

Comment:
Not sure about the need for an underscore, I don't see why this shouldn't be called ``StatisticalElement`` even if it is a base class. I can imagine ``isinstance(foo, StatisticalElement)`` being useful.